### PR TITLE
Fix #17342 - Fix indentation / markdown formatting.

### LINKED ISSUE #17342: Issue with k8s.io/docs/tasks/administer-cluster/namespaces/
**This is a Bug Report**

**Problem:**
Ordered list is broken. Each item starts "1".
> 1 . Understand the default namespace
> ...
> 1 . Create new namespaces
> ...
> 1 . Create pods in each namespace

https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#subdividing-your-cluster-using-kubernetes-namespaces

**Proposed Solution:**
> 1. Understand the default namespace
> ...
> 1. Create new namespaces
> ...
> 1. Create pods in each namespace

**Page to Update:**
https://kubernetes.io/docs/tasks/administer-cluster/namespaces/

### PR DESCRIPTION
**Pull request by miteshskj**
Fixes #17342

### CODE REVIEW

**Comment by daminisatya** [comment]
@sftim

**Comment by miteshskj** [comment]
@lojies  - The code snippet starts from 1st char, breaking the alignment and the numbering sequence. The sequence breaks if any line starts before 5th char.
Also, that code (yaml file) is inconsistent with the following command (command is using the yaml file via http instead of local).

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 1]
```diff
@@ -111,213 +111,209 @@ More information on `finalizers` can be found in the namespace [design doc](http
 
 1. Delete a namespace with
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md]: Does this need to stay as a numbered list? I think for this particular section I'd remove the list numbering and leave the indentation (down to line 122) as-in.

Comment (miteshskj) [reply content/en/docs/tasks/administer-cluster/namespaces.md]: Thanks @sftim  - We do not need a numbered list for this item. Removed as suggested.

**Comment by sftim** [comment]
Also
/lgtm

**Comment by sftim** [comment]
@miteshskj could you rebase this against _master_, squashing your commits?
See https://kubernetes.io/docs/contribute/intermediate/#work-from-a-local-clone for tips

**Comment by miteshskj** [comment]
@sftim  - Thanks for the review and suggestions. My branch got messed up, so created a new PR - https://github.com/kubernetes/website/pull/17465

**Pull request closed without merging**